src/shared/components/query/styles.module.scss

```diff
@@ -344,7 +349,8 @@ div.SelectedStudiesWindow {
     width: 555px;
     resize: vertical;
     padding: 5px;
-    border-color:$borderColor;
+    border: 1px solid $borderColor;
+    border-radius: 2px;

```

Comment:
Without these 2 lines, case set selector text area looks different in IE and Firefox.